tests/SendGrid.Extensions.DependencyInjection.Tests/ServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace SendGrid.Extensions.DependencyInjection.Tests
+{
+    public class ServiceCollectionExtensionsTests
+    {
+        [Fact]
+        public void TestAddSendGridWithoutApiKey()
+        {
+            // Arrange
+            var services = new ServiceCollection().AddSendGrid(options => { }).Services.BuildServiceProvider();
+
+            // Act && Assert
+            Assert.Throws<ArgumentNullException>(() => services.GetRequiredService<ISendGridClient>());
+        }
+
+        [Fact]
+        public void TestAddSendGridReturnHttpClientBuilder()
+        {
+            // Arrange
+            var collection = new ServiceCollection();
+
+            // Act
+            var builder = collection.AddSendGrid(options =>
+                options.ApiKey = Environment.GetEnvironmentVariable("SENDGRID_API_KEY") ?? "SENDGRID_API_KEY");
```

Comment:
Wouldn't it be more efficient to forego the `Environment.GetEnvironmentVariable` call entirely, since the value possibly retrieved from it isn't being used anyway?

Reply:
Thanks 👌